### PR TITLE
Feature/betting

### PR DESCRIPTION
**Pull request by TommyMedina**

### CODE REVIEW

**Review by grant-warkentin — APPROVED** [review]
Besides adding comments, it looks good.

File: test/test_betting.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
+from testing_base import *
+
```
Comment (php-121) [inline test/test_betting.py:2]: Nice test, I like that you have a bunch of test cases

File: source/CardGames.py [hunk 2]
```diff
@@ -84,6 +84,31 @@ def getCard( suit, value):
     if card == my_card:
       return card
   return None
+class betting_box:
+  def __init__(self, money: int = 0):
+    self.wager = money
+    self.betters= []
+
+  def bet(self, bet, player: bool = False):
```
Comment (php-121) [inline source/CardGames.py:92]: I think you should add a case for no player or no bet.

**Review by php-121 — COMMENTED** [review]
LGTM!

**Pull request merged**